components/google_sheets/google_sheets.app.js

```diff
@@ -18,16 +18,13 @@ module.exports = {
       label: "Worksheets to watch for changes",
       async options({ sheetId }) {
         const { sheets } = await this.getSpreadsheet(sheetId);
-        return sheets.map(sheet => {
-          const {
-            title: label,
-            sheetId: value,
-          } = sheet.properties;
-          return {
-            label,
-            value,
-          };
-        });
+        return sheets
+          .map(({ properties }) => properties)
+          .filter(({ sheetType }) => sheetType === 'GRID')
```

Comment:
Probably worth a comment here on what the "GRID" sheets actually mean (is there a link to docs?)

Reply:
Good idea. Done.